web-app/src/users/api.admin.users.pipes.ts

```diff
@@ -0,0 +1,12 @@
+import { PipeTransform, Injectable, ArgumentMetadata } from '@nestjs/common';
+
+@Injectable()
+export class SimpleJsonParsePipe<T> implements PipeTransform<any, T> {
```

Comment:
Why duplicate this pipe? We should rather move the existing pipe to a common folder, like a 'utils' folder